### ISSUE TITLE
Add Issue Template

### ISSUE THREAD

**Issue by mlodic** [intelowlproject/IntelOwl#854]
We need to get inspiration from another famous project and add it to IntelOwl

**Comment by 0x0elliot** [comment]
For reference, We can take a look at [CTFd](https://github.com/CTFd/CTFd/issues/new). Their template looks like this:

```
<!--
If this is a bug report please fill out the template below.

If this is a feature request please describe the behavior that you'd like to see.
-->

**Environment**:

- CTFd Version/Commit:
- Operating System:
- Web Browser and Version:

**What happened?**

**What did you expect to happen?**

**How to reproduce your issue**

**Any associated stack traces or error logs**
```

Another cool thing that I noticed was [MobSf](https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/new/choose)'s issue tab:

![Screenshot 2022-02-01 at 6 57 39 PM](https://user-images.githubusercontent.com/60684641/151977010-e77df9c2-450c-43f0-8c4d-1dd0e1ec5dcf.png)

I am not sure how they put this together but I felt like this was perfect considering the future of IntelOwl that I have understood. All three sections have a pretty comprehensively put-together structure as well.

**Comment by fear-the-reaper** [comment]
Can anyone tell me what needs to be done I need to make a markdown template or something?

**Comment by mlodic** [comment]
just check how the issue templates are created like that repo and propose a PR

**Comment by fear-the-reaper** [comment]
I'm having trouble with making a new branch on the develop branch. whenever I type `git checkout -b issue_template develop`
it gives out 
![image](https://user-images.githubusercontent.com/46054431/155492810-9faa4024-570c-42e8-96fa-c07cc3599fd7.png)

**Comment by fear-the-reaper** [comment]
Never mind fixed.

**Comment by fear-the-reaper** [comment]
Do I need to run the github CI tests? As I haven't written any code just markdown also saw how to get the issues tab as well.

**Comment by fear-the-reaper** [comment]
Alright the tests passed but I can't push my code for some reason. First github said to do the `git push --set-upstream origin issue_template` it didn't work. 
![image](https://user-images.githubusercontent.com/46054431/155545963-e783aad1-fe44-46ec-9154-6dacf9aaa796.png)

**Comment by mlodic** [comment]
you need to create a fork of the main repository and make a PR from there.

**Comment by fear-the-reaper** [comment]
Oh! will do that quick question to enable the issue tab feature you need to enable issues down in settings should I do that in my fork and once (hopefully) when my PR is accepted will the changes be added or is this done through a maintainer. I also made a new analyzer template as well.

**Comment by mlodic** [comment]
fixed with v3.3.1
